test/test_routes.py

```diff
@@ -824,7 +824,7 @@ def identity(x, y, request: gr.Request):
         response = client.post(f"{API_PREFIX}/api/chat/", json={"data": ["test", None]})
         assert response.status_code == 200
         output = dict(response.json())
-        assert output["data"] == ["test", None]
+        assert output["data"] == [None, "test"]
```

Comment:
Now the state variable is first so the None appears first, but this should not have any impact since the Client filters state variables out anyways.